relayer-cli/src/commands/tx/connection.rs

```diff
@@ -6,8 +6,8 @@ use ibc::ics24_host::identifier::{ClientId, ConnectionId};
 
 use relayer::config::Config;
 use relayer::tx::connection::{
-    build_conn_init_and_send, build_conn_try_and_send, ConnectionOpenInitOptions,
-    ConnectionOpenTryOptions,
+    build_conn_ack_and_send, build_conn_confirm_and_send, build_conn_init_and_send,
+    build_conn_try_and_send, ConnectionOpenInitOptions, ConnectionOpenOptions,
```

Comment:
I noticed that we use `ConnectionOpenOptions` for most conn commands such as try, ack, confirm, with the exception of init, which has a dedicated one `ConnectionOpenInitOptions`. The two structs `ConnectionOpenInitOptions` and `ConnectionOpenOptions` are identical, however, so maybe we can delete the init-specific one.

Reply:
they are not identical, the init one has an optional counterparty (or source) connection id. I could have the other general one use the option but then every time it is used we need to assert on `.is_some()`